lib/setup.js

```diff
@@ -36,6 +45,12 @@ var constructResponse = function (server) {
           path: route.path,
           plugin: route.public.realm.plugin || null
         };
+      }).sort(function (a, b) {
+        var pathCompare = a.path.localeCompare(b.path);
+        if (pathCompare === 0) {
+          return methodSort[a.method] - methodSort[b.method];
```

Comment:
You're already uppercasing the methods on line 44. Also, hapi throws if you try to use a method that doesn't exist.